sqlite-db/src/test_events/contract_setup_completed.json

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
❓ Did we forget to delete the originals? I'm surprised these appear as new files.

Reply:
I had to duplicate it because this is needed for the tests which were moved to projection.rs

Reply:
I opted for moving it and doing file traversing to find the file.